src/main/java/io/jenkins/plugins/ml/IPythonKernelInterpreter.java

```diff
@@ -96,6 +93,11 @@ public void start() throws InterpreterException {
 
     @Override
     public void shutdown() throws InterpreterException {
+        try {
+            Thread.sleep(5000);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
```

Comment:
Why is this necessary? I couldn't find any comments here, nor in Javadocs or in any of the JIRA's mentioned here?

Reply:
Oops !. I was trying to solve the null pointer issue by giving some time to close the interpreter. Forgot to remove that. will remove it